<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoReciteDialog</class>
 <widget class="QDialog" name="MoReciteDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QToolButton" name="searchBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/search.png</normaloff>:/images/search.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="bookBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/book.png</normaloff>:/images/book.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="statisticBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/statistic.png</normaloff>:/images/statistic.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="personalBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/user.png</normaloff>:/images/user.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="wordLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>240</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>word</string>
   </property>
  </widget>
  <widget class="QLabel" name="phonogramLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>95</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>phonogram</string>
   </property>
  </widget>
  <widget class="QLabel" name="translationLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>260</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>translation</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="leftBtn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="middleBtn">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>210</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="rightBtn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>210</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="wordLineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>95</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="logoLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>logo</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
